modules/odc-customizing-available-cluster-roles-using-developer-perspective.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// applications/projects/working-with-projects.adoc
+
+[id="odc-customizing-available-cluster-roles-using-developer-perspective_{context}"]
+= Customizing the available cluster roles using the Developer perspective
+
+You can customize cluster role's name of existing projects by editing the YAML code in the `console.openshift.io/display-name: v1`.
+
+.Procedure
+Customize the cluster role options of all projects that are shown in the *Project Access* tab:
+
+. In the *Search* view, select the *Resources* drop down and search "*Console*".
+. Select the *Console `operator.openshift.io/v1`*.
++
+.Searching Console resource
+image::odc_cluster_console.png[]
+. Select *cluster* under the *Name* list.
+. Navigate to *YAML* tab to view and edit the YAML code.
+. In the YAML code under `spec` add or edit the list of `availableClusterRoles`.
++
+[source,yaml]
+----
+spec:
+  customization:
+    projectAccess:
+      availableClusterRoles:
+      - admin
+      - edit
+      - view
+----
++
+For example, you can remove `view` role from the list to show `admin` and `edit` roles only.
+. Click *Save*.
+
+Verify the changes in the *Role* drop down under the *Project Access* tab of *Project* view.
+
+.Project roles
```

Comment:
I don't see the project roles within this procedure in the preview link.

Reply:
Those are there in the screenshot. Let us check again.